src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaCloudformation.tsx

```diff
@@ -185,64 +200,83 @@ export default class AwsLambdaCloudformation extends React.Component<Props, Stat
       accountNumberError,
       submitting,
       awsExternalId,
+      showInputs,
     } = this.state;
     return (
       <React.Fragment>
         <HeaderWithHelp docsUrl="https://docs.sentry.io/product/integrations/aws-lambda/" />
         <StyledList symbol="colored-numeric" initialCounterValue={initialStepNumber}>
           <ListItem>
-            <ExternalLink
+            <h3>{t("Add Sentry's CloudFormation")}</h3>
+            <StyledButton
+              priority="primary"
               onClick={this.trackOpenCloudFormation}
+              external
               href={this.cloudformationUrl}
             >
-              <h3>{t("Add Sentry's CloudFormation to your AWS")}</h3>
-            </ExternalLink>
-            <p>{t('After the stack has been created, continue to the next step')}</p>
-          </ListItem>
-          <ListItem>
-            <h4>{t('Add AWS Account Information')}</h4>
-            <TextField
-              name="accountNumber"
-              value={accountNumber}
-              onChange={this.handleChangeArn}
-              onBlur={this.validateAccountNumber}
-              error={accountNumberError}
-              inline={false}
-              stacked
-              label={t('AWS Account Number')}
-              showHelpInTooltip
-              help={t(
-                'Your account number can be found on the right side of the header in AWS'
-              )}
-            />
-            <SelectField
-              name="region"
-              value={region}
-              onChange={this.hanldeChangeRegion}
-              options={this.regionOptions}
-              allowClear={false}
-              inline={false}
-              stacked
-              label={t('AWS Region')}
-              showHelpInTooltip
-              help={t(
-                'Your current region can be found on the right side of the header in AWS'
-              )}
-            />
-            <TextField
-              name="awsExternalId"
-              value={awsExternalId}
-              onChange={this.handleChangeExternalId}
-              inline={false}
-              stacked
-              error={awsExternalId ? '' : t('External ID Required')}
-              label={t('External ID')}
-              showHelpInTooltip
-              help={t(
-                'Do not edit unless you are copying from a previously created CloudFormation stack'
-              )}
-            />
+              {t('Go to AWS')}
+            </StyledButton>
+            {!showInputs && (
+              <React.Fragment>
+                <p>
+                  {t(
+                    "Once you've created Sentry's CloudFormation stack (or if you already have one) press the button below to continue."
```

Comment:
Nit: I think `click the button` is more intuitive

Reply:
@NisanthanNanthakumar normally I would agree but there are two buttons so I wanted to be specific